<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportReadsDialog</class>
 <widget class="QDialog" name="ExportReadsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>123</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export Reads</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="filepathLabel">
       <property name="text">
        <string>Export to file</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="filepathLineEdit"/>
     </item>
     <item row="0" column="2">
      <widget class="QToolButton" name="filepathToolButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="documentFormatLabel">
       <property name="text">
        <string>File format</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QComboBox" name="documentFormatComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="addToProjectCheckBox">
     <property name="text">
      <string>Add to project</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
